packages/help-center/src/components/help-center-header.scss

```
@@ -59,7 +59,7 @@
 				display: inline-block;
 				margin-left: 8px;
 				padding: 2px 8px;
-				background: var(--studio-pink-50);
+				background: var(--studio-wordpress-blue);
```

```suggestion
				background: var(--color-masterbar-unread-dot-background);
```

Could we use this instead? To match what the masterbar notification dot is?

---

Good idea! I just made this update.